### PR TITLE
Gridded timeseries_v2

### PR DESCRIPTION
**Pull request by diodon**
gridded time series product with amend in the documentation, code and attributes

### CODE REVIEW

File: aodntools/timeseries_products/gridded_timeseries.py [hunk 1]
```diff
@@ -209,7 +209,7 @@ def grid_variable(file_name, VoI, depth_bins=None, max_separation=50, depth_bins
     ## create the output file name and write the aggregated product as netCDF
     facility_code = TStools.get_facility_code(file_name)
     data_code = TStools.get_data_code(VoI) + 'Z'
-    product_type = 'gridded-timeseries'
+    product_type = 'timeSeriesProfile'
```
Comment (mhidas) [inline aodntools/timeseries_products/gridded_timeseries.py]: Sorry, this is not what I meant in my email. 'gridded-timeseries' is fine in the file name.
It's the `feature_type` global attribute that should be 'timeSeriesProfile' (or removed, which might be necessary to pass the compliance checks)

Comment (diodon) [reply aodntools/timeseries_products/gridded_timeseries.py]: fixed. It is featureType without the underscore, coming from the original files and passed to the hourly. It is quite weird as all the other two-word global attrs are underscore separated.

File: aodntools/timeseries_products/gridded_timeseries.py [hunk 2]
```diff
@@ -115,17 +115,19 @@ def grid_variable(file_name, VoI, depth_bins=None, max_separation=50, depth_bins
     ## create empty containers
     time_0 = pd.to_datetime('1950-01-01T00:00:00')
     time_min = nc.TIME.values.min()
-    depth_bin_len = len(depth_bins)
-    VoI_binned = xr.DataArray(np.full((depth_bin_len, 1), np.nan), coords=[depth_bins, [time_0]],
+    depth_bin_len = len(depth_bins)\
```
[the remark this answers is not in the document]
Comment (diodon) [reply aodntools/timeseries_products/gridded_timeseries.py]: fixed

File: aodntools/timeseries_products/gridded_timeseries.py (hunk 1 above)
Comment (mhidas) [reply aodntools/timeseries_products/gridded_timeseries.py]: Yes, it's inconsistent, but `featureType` is what CF expects (http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html#featureType)

**Pull request merged**